students/kegan/session01/2017 10 03 Session 1.txt

```diff
@@ -0,0 +1,2 @@
+questions:
+1) any advantage to using del versus just reassigning or ignoring the variable?
```

Comment:
del is there if you need it, but you are quite right — it’s actually pretty rare to need it.

Most often you can re-use the name as you say, or simple ignore it — it will usually go out of scope soon anyway. — at the end of a function, for instance.”